packages/grid/_modules_/grid/components/toolbar/GridFilterToolbarButton.tsx

```diff
@@ -63,6 +83,8 @@ export const GridFilterToolbarButton: React.FC<{}> = () => {
   return (
     <Tooltip title={tooltipContentNode} enterDelay={1000}>
       <Button
+        ref={ref}
+        {...props}
```

Comment:
The Tooltip is the root, not the button. In practice, the Tooltip forwards the extra props and the ref to the Button. So I think that we can do

```suggestion
    <Tooltip title={tooltipContentNode} enterDelay={1000} ref={ref} {...props}>
      <Button
```

Reply:
But maybe the current approach would be simpler for TypeScript

Reply:
I was thinking the same/ The thing is that looking at how the button is called makes sense for the props to be on the button itself. Also if you look at what is in the DOM the button is the root element.